tst/EnergyPlus/unit/AdvancedAFN.unit.cc

```diff
@@ -116,35 +116,36 @@ TEST_F(EnergyPlusFixture, AirflowNetwork_AdvancedTest_Test1)
     state->afn->OccupantVentilationControl(1).ComfortHighTempCurveNum = 2;
 
     state->dataCurveManager->NumCurves = 2;
-    state->dataCurveManager->PerfCurve.allocate(state->dataCurveManager->NumCurves);
+    for (int curveIndex = 1; curveIndex <= 2; curveIndex++)
+        state->dataCurveManager->PerfCurve.push_back(new EnergyPlus::Curve::Curve);
```

Comment:
Are unit tests not in the `EnergyPlus` namespace?

Reply:
There was something odd on Windows/Visual Studio causing the namespace issue.  In this particular case, I don't think it was needed.  I can back it out and verify on Windows before pushing.